addons/explosives/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,44 @@
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"
+
+["ACE Explosives", QGVAR(openCellphone), LLSTRING(cellphone_displayName),
+{
+    if (
+        !('ACE_Cellphone' in (items ACE_player)) ||
+        {ACE_player getVariable [QEGVAR(captives,isSurrendering), false]} ||
+        {ACE_player getVariable [QEGVAR(captives,isHandcuffed), false]}
+    ) exitWith {};
```

Comment:
```suggestion
    if !(
        ("ACE_Cellphone" in (items ACE_player) &&
        {[ACE_player, objNull, ["isNotSwimming", "isNotInside", "isNotSitting"]] call EFUNC(common,canInteractWith)}
    ) exitWith {};
```

Reply:
Use `EFUNC(common,uniqueItems)` instead of `items`, as it's cached.